### DISCUSSION TITLE
Wails --detached ?

### DISCUSSION THREAD

**Discussion by radoslaw-szpiech** [wailsapp/wails#2364] [Q&A]
Is there a way to run wails in detached mode?

My use case:
I'm using `wails dev` as one of CI step. I'd like to have it built and run, and then moved to background when it's done successfully. As for now my workaround is to run `wails dev -noreload &` and then listen to port with timeout, but I'm loosing possibility to see any errors during build time.

**Comment by radoslaw-szpiech** [comment]
Adding some context here - https://github.com/vegaprotocol/vegawallet-desktop/blob/develop/.github/workflows/test-frontend.yml#L62

**Reply by airtonix** [reply]
file is 404.

instead of linking files at github to a branch, use the commit hash.
